closed/autoconf/custom-hook.m4

```diff
@@ -62,18 +62,7 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
       fi
     ],
     [
-      case "$OPENJ9_PLATFORM_CODE" in
-        ap64|mz64|oa64|or64|wa64|xa64|xl64|xr64|xz64)
-          if test "x$COMPILE_TYPE" != xcross ; then
-            with_cmake=cmake
-          else
-            with_cmake=no
-          fi
-          ;;
-        *)
-          with_cmake=no
-          ;;
-      esac
+      with_cmake=cmake
```

Comment:
This should be simplified more than it is. As it stands, we are testing if `--with-cmake` is yes or not set, and then setting cmake, but also setting cmake in the else case.

Reply:
Or to put it another way, this change breaks `--with-cmake=no`

Reply:
No, I don't think so. The third and fourth arguments to `AC_ARG_WITH` are for when `--with-cmake` is used and for when it is not, respectively. `--with-cmake=no` (and `--without-cmake`) will be handled by the third argument which will leave `with_cmake=no` alone.